src/hotspot/share/interpreter/bytecode.hpp

```diff
@@ -71,11 +71,11 @@ class Bytecode: public StackObj {
   Bytecodes::Code invoke_code() const            { return (code() == Bytecodes::_invokehandle) ? code() : java_code(); }
 
   // Static functions for parsing bytecodes in place.
-  int get_index_u1(Bytecodes::Code bc) const {
+  u1 get_index_u1(Bytecodes::Code bc) const {
```

Comment:
We have a mix of u1 and jubyte in these functions. That looks odd. I am not sure whether we should fix that.

Reply:
We've been trying to get away from the j<int> types but they persist in old code. Most of the classfile code prefers to use u1, u2 etc, so that's why I picked that especially for the function that's called get_index_u1.  I changed a couple of nearby jubyte to u1.